core/src/test/scala/unit/kafka/security/auth/ZkAuthorizationTest.scala

```diff
@@ -197,7 +198,7 @@ class ZkAuthorizationTest extends ZooKeeperTestHarness with Logging{
       case true => "secure"
       case false => "unsecure"
     }
-    ZkSecurityMigrator.run(Array(s"--zookeeper.acl=$secureOpt", s"--zookeeper.connect=$zkConnect"))
+    ZkSecurityMigrator.run(Array(s"--zookeeper.acl=$secureOpt", s"--zookeeper.connect=$zkUrl"))
     info("Done with migration")
     for (path <- secondZk.securePersistentZkPaths) {
```

Comment:
@ewencp good suggestion, I've changed accordingly. let me know if it looks good to you.